sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/http/ReactorNettyClient.java

```diff
@@ -172,31 +172,26 @@ public HttpHeaders headers() {
 
         @Override
         public Flux<ByteBuf> body() {
-            return bodyIntern().doFinally(s -> this.close());
+            return bodyIntern();
         }
 
         @Override
         public Mono<byte[]> bodyAsByteArray() {
-            return bodyIntern().aggregate().asByteArray().doFinally(s -> this.close());
+            return bodyIntern().aggregate().asByteArray();
         }
 
         @Override
         public Mono<String> bodyAsString() {
-            return bodyIntern().aggregate().asString().doFinally(s -> this.close());
+            return bodyIntern().aggregate().asString();
         }
 
         @Override
         public Mono<String> bodyAsString(Charset charset) {
-            return bodyIntern().aggregate().asString(charset).doFinally(s -> this.close());
+            return bodyIntern().aggregate().asString(charset);
         }
 
         @Override
         public void close() {
-            if (reactorNettyConnection.channel().eventLoop().inEventLoop()) {
-                reactorNettyConnection.dispose();
-            } else {
-                reactorNettyConnection.channel().eventLoop().execute(reactorNettyConnection::dispose);
-            }
         }
```

Comment:
Might the method override be removed now that the body of close is empty?

Reply:
Makes sense, will remove the empty method.

Reply:
Done